src/main/java/org/eln2/mc/client/input/Input.kt

```diff
@@ -0,0 +1,56 @@
+package org.eln2.mc.client.input
+import com.mojang.blaze3d.platform.InputConstants
+import net.minecraft.client.KeyMapping
+import net.minecraft.client.Minecraft
+import net.minecraftforge.client.ClientRegistry
+import net.minecraftforge.client.event.InputEvent
+import net.minecraftforge.client.settings.KeyConflictContext
+import net.minecraftforge.eventbus.api.SubscribeEvent
+import net.minecraftforge.fml.common.Mod
+import org.apache.logging.log4j.LogManager
+import org.eln2.mc.client.gui.CellInfo
+import org.eln2.mc.client.gui.PlotterScreen
+import org.eln2.mc.client.overlay.plotter.PlotterOverlay
+import org.lwjgl.glfw.GLFW
+
+//String description,
+// net.minecraftforge.client.settings.IKeyConflictContext keyConflictContext,
+// final InputConstants.Type inputType,
+// final int keyCode,
+// String category
```

Comment:
What is this documenting? If it's the KeyMapping, might as well add a Javadoc comment for it